docs/architecture/microservices/architect-microservice-container-applications/microservice-based-composite-ui-shape-layout.md

```diff
@@ -36,9 +36,6 @@ However, we encourage you to use the following references to learn more about co
 - **Composite UI using ASP.NET (Particular's Workshop)** \
   <https://github.com/Particular/Workshop/tree/master/demos/asp-net-core>
 
-- **Ruben Oostinga. The Monolithic Frontend in the Microservices Architecture** \
-  <https://blog.xebia.com/the-monolithic-frontend-in-the-microservices-architecture/>
```

Comment:
Instead of removing, we should probably change it to https://xebia.com/blog/the-monolithic-frontend-in-the-microservices-architecture/

Reply:
Yes, agreed.

Reply:
Added a suggestion to make it easier for you to just commit the change.